_posts/2018-09-03-getting-started/getting-started.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Getting started with deep learning in R"
+description: |
+  More and more fields are profiting from the use of deep learning, and with the R keras, tensorflow and related packages, you can do deep learning directly in R. In this post, we want to give some orientation as to how to best get started.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2018gettingstarted
+date: 09-03-2018
+categories:
+  - Keras
+output:
+  radix::radix_article:
+    self_contained: false
+preview: images/digits.png
+---
+
+
+There are good reasons to get into deep learning: Deep learning has been outperforming the respective "classical" techniques in areas like image recognition and natural language processing for a while now, and it brings interesting insights even to the analysis of tabular data. For many R users potentially interested in deep learning, the hurdle is not an anticipation of a bit of math creeping up (as many have a background in statistics or empirical sciences). The question is how to get started in an efficient way.
+
+This post will give an overview of materials that should be useful. And if you don't have that background in statistics or similar: Further down, we will also present a few helpful resources to catch up with "the math".
+
+## Keras
+
+The easiest way to get started is using the Keras API. It is a high-level, declarative (in feel) way of specifying a model, training and testing it, originally developed in [Python](http://keras.io) by Francois Chollet and ported to R by JJ Allaire.
+
+Check out the tutorials on the [Keras website](https://tensorflow.rstudio.com/keras/): They introduce basic tasks like classification and regression, as well as basic workflow elements like saving and restoring models, or assessing model performance.
+
+Once you've seen the basics, the website also has more advanced information as to implementing custom logic, monitoring and tuning, as well as using and adapting pre-trained models.
+
+If you don't yet have prior experience in deep learning, you might want a bit more background. [Deep Learning with R](https://www.amazon.com/Deep-Learning-R-Francois-Chollet/dp/161729554X) by Francois Chollet and JJ Allaire is an excellent introduction to deep learning that presents the high-level concepts in a very intellegible way.
```

Comment:
"If you don't yet have prior experience in deep learning"

I'd say the tutorials are also for those w/o prior experience so we might just lead with "If you want a bit more background..."

Reply:
oops, yeah that sentence I wrote was totally illogical!!